schema/omega-services.async.api.yaml

```diff
@@ -150,6 +228,58 @@ components:
         items:
           $ref: '#/components/schemas/LegalStatusFull'
 
+    AgentSummaryList:
+      messageId: ODLISAGT001
```

Comment:
It is not entirely clear to me what the purpose of having a `messageId` defined for the reply message is. I had initially thought this was intended to work in the same way as it does in the API services - i.e. to identify the message payload for unmarshalling - however this is not needed currently in the front end as we have a map which binds the request and reply via the message ID / correlation ID so we already know what is expected. We could use this as extra layer of validation, i.e. is the `messageId` as expected? But even this seems superflous as we will know straight away if we can't parse the payload.